Document 500 project per organization limit for Serverless (#7332)
elastic/docs-content: deploy-manage/deploy/_snippets/create-serverless-project-intro.md

:::{note}
If your organization restricts outbound browser traffic with a firewall, proxy, or secure web gateway, make sure users can access `kibana.estccdn.com` and `cloud.elastic.co`. If these domains are blocked, {{kib}} can load as a blank page in {{serverless-short}} projects. For more information, refer to [Browser access requirements](/deploy-manage/deploy/elastic-cloud.md#browser-access).
:::

:::{note}

> By default, {{ecloud}} organizations are limited to 500 {{serverless-short}} projects. If you reach this limit, you'll get an error when trying to create additional projects. This is a soft limit, so if you need more projects, [contact Elastic Support](/troubleshoot/index.md#contact-us) to request an increase.

Reviewer: ```suggestion
{{ecloud}} organizations are limited to 500 {{serverless-short}} projects. If you reach this limit, you'll get an error when trying to create additional projects. These limits are adjustable and can be increased by request; if you need more projects, [contact Elastic Support](/troubleshoot/index.md#contact-us) to request an increase.
```

Reviewer: Just updating the language to match how we talk about this on the other page.

> By default, {{ecloud}} organizations are limited to 500 {{serverless-short}} projects. If you reach this limit, you'll get an error when trying to create additional projects. This is a soft limit, so if you need more projects, [contact Elastic Support](/troubleshoot/index.md#contact-us) to request an increase.

Reviewer 2: I will say "adjustable limit" to align with the terminology we use in the other page (Line 179 of Differences from other ES offerings). 

Also, should we use the same language from Line 179 as it is to keep info consistent across pages.

>These limits are adjustable and can be increased by request. To request a limit increase, [open a support case](/troubleshoot/index.md#contact-us), and include your preferred new value and a brief description of your use case.

Author: I've applied @yetanothertw's suggestion as she raised the same concern. Let us know if that's also valid, using the `adjustable` term in the second statement.

update: I've used finallly `adjustable limit` and removed the `This limit is adjustable` in the final clause (cc: @yetanothertw , as I've updated your previous suggestion).

> {{ecloud}} organizations are limited to 500 {{serverless-short}} projects. If you reach this limit, you'll get an error when trying to create additional projects. These limits are adjustable and can be increased by request; if you need more projects, [contact Elastic Support](/troubleshoot/index.md#contact-us) to request an increase.

Author: ```suggestion
{{ecloud}} organizations are limited to 500 {{serverless-short}} projects. If you reach this limit, you'll get an error when trying to create additional projects. These limits are adjustable and can be increased by request. If you need more projects, [contact Elastic Support](/troubleshoot/index.md#contact-us) to request an increase.
```

> {{ecloud}} organizations are limited to 500 {{serverless-short}} projects. If you reach this limit, you'll get an error when trying to create additional projects. These limits are adjustable and can be increased by request. If you need more projects, [contact Elastic Support](/troubleshoot/index.md#contact-us) to request an increase.

Author: ```suggestion
{{ecloud}} organizations have an adjustable limit of 500 {{serverless-short}} projects. If you reach this limit, you'll get an error when trying to create additional projects. To request an increase, [contact Elastic Support](/troubleshoot/index.md#contact-us).
```
